### PR TITLE
Implemented FTSwish Activation Function

### PR DESCRIPTION
**Pull request by mayank-root**
This PR Implements the FTSwish Activation Layer.

### CODE REVIEW

**Comment by mayank-root** [comment]
@zoq please review.

File: src/mlpack/methods/ann/layer/ftswish_impl.hpp [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+{
+  #pragma omp for
+  for (size_t i = 0; i < (size_t) input.n_elem; ++i)
+  {
+    if (input(i) >= 0)
+    {
+      const double f_x = input(i) / (1 + std::exp(-input(i)));
+      const double sigmoid_x = 1 / (1 + std::exp(-input(i)));
+     
+      g(i) = gy(i) * (sigmoid_x * (1 - f_x) + f_x);
```
Comment (zoq) [inline src/mlpack/methods/ann/layer/ftswish_impl.hpp:97]: Please use camel casing for all names, see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#naming-conventions for details.

File: src/mlpack/methods/ann/layer/ftswish_impl.hpp [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+template<typename MatType>
+void FTSwishType<MatType>::Backward(
+    const MatType& input, const MatType& gy, MatType& g)
+{
+  #pragma omp for
+  for (size_t i = 0; i < (size_t) input.n_elem; ++i)
+  {
+    if (input(i) >= 0)
+    {
+      const double f_x = input(i) / (1 + std::exp(-input(i)));
```
Comment (zoq) [inline src/mlpack/methods/ann/layer/ftswish_impl.hpp:94]: See comment below.

File: src/mlpack/tests/ann/layer/ftswish.cpp [hunk 3]
```diff
@@ -0,0 +1,62 @@
+/*
+ * @file tests/ann/layer/hard_tanh.cpp
```
Comment (zoq) [inline src/mlpack/tests/ann/layer/ftswish.cpp:2]:
```suggestion
 * @file tests/ann/layer/ftswish.cpp
```

File: src/mlpack/methods/ann/layer/ftswish_impl.hpp (hunk 1 above)
Comment (mayank-root) [reply src/mlpack/methods/ann/layer/ftswish_impl.hpp]: Is it alright now?

**Review by zoq — APPROVED** [review]
No further comments from my side, thanks for putting this together.

**Review by Abhinav-Anand-Lambdatest — APPROVED** [review]
Looks good.

**Comment by zoq** [comment]
Thanks again for putting this together.

**Pull request merged**